hr_timesheet_retroactive_cost/wizards/hr_employee_timesheet_cost_wizard.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022-2024 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class HrEmployeeTimesheetCost(models.TransientModel):
+    _name = "hr.employee.timesheet.cost.wizard"
+    _description = "Employee timesheet cost wizard"
+
+    employee_id = fields.Many2one(comodel_name="hr.employee")
+    currency_id = fields.Many2one("res.currency")
+    timesheet_cost = fields.Monetary(currency_field="currency_id")
+    retroactive_cost_date = fields.Date(
+        help="Change de timesheet cost from this date to now.",
+        default=fields.Datetime.now,
+    )
```

Comment:
**issue:** all fields should be required. Add ondelete clauses too, so wizards don't block deletion of real records. Add strings too:

```suggestion
    employee_id = fields.Many2one(comodel_name="hr.employee", string="Employee", ondelete="cascade", required=True)
    currency_id = fields.Many2one("res.currency", string="Currency", ondelete="cascade", required=True)
    timesheet_cost = fields.Monetary(currency_field="currency_id", required=True)
    retroactive_cost_date = fields.Date(
        required=True,
        help="Change timesheet cost from this date onwards.",
        default=fields.Datetime.now,
    )
```